src/System.Private.CoreLib/src/System/ThrowHelper.cs

```diff
@@ -392,6 +405,27 @@ internal static void ThrowNotSupportedExceptionIfNonNumericType<T>()
                 throw new NotSupportedException(SR.Arg_TypeNotSupported);
             }
         }
+
+        //
+        // Enable use of ThrowHelper from TryFormat() routines without introducing dozens of non-code-coveraged "bytesWritten = 0; return false" boilerplate.
+        //
+        public static bool TryFormatThrowFormatException(out int bytesWritten)
+        {
+            bytesWritten = 0;
+            ThrowHelper.ThrowFormatException_BadFormatSpecifier();
+            return false;
+        }
+
+        //
+        // Enable use of ThrowHelper from TryParse() routines without introducing dozens of non-code-coveraged "value= default; bytesConsumed = 0; return false" boilerplate.
+        //
+        public static bool TryParseThrowFormatException<T>(out T value, out int bytesConsumed)
```

Comment:
This helper is very Utf8Formatter specific. It would be better for it to live on Utf8Formatter.